sample/build.gradle

```diff
@@ -1,13 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
-    buildToolsVersion "30.0.2"
+    compileSdkVersion 30
 
     defaultConfig {
         applicationId "com.tumblr.permissme.sample"
         minSdkVersion 16
-        targetSdkVersion 28
+        targetSdkVersion 30
```

Comment:
Should we target 31? We are working on upgrading this on the main repo, and the main blocker we have is the lack of Jdk 11 on Jenkins (which is also required for AGP 7.x), so we can do both at the same time

Reply:
same for `compileSdkVersion`

Reply:
@obsantos Sounds good, addressed it in 3b617ac96a3f12c22a63d1c208293e5c46f6b75c.